bin/install_travis.sh

```diff
@@ -12,8 +12,13 @@ if [[ "${WITH_SAGE}" != "yes" ]]; then
     conda config --set always_yes yes --set changeps1 no;
     conda update -q conda;
     conda info -a;
-
-    conda create -q -n test-environment python="${PYTHON_VERSION}" pip cython sympy nose pytest;
+    CONDA_PKGS="pip cython sympy nose pytest"
+    if [[ "${SKIP_NUMPY}" == "yes" ]]; then
```

Comment:
Yes. We already have tons the `WITH_*` options, so we just add another one. I think it's easier to maintain it that way --- for each test you simply enable what you need.